html/webappapis/animation-frames/same-dispatch-time.html

```diff
@@ -24,7 +24,7 @@
           assert_not_equals(b, 0);
           assert_equals(a, b);
           t.done();
-        });
+        }));
```

Comment:
@lukebjerring here's another thing that I noticed only because I went hunting through the results to sniff out problem. I'll stop bugging with with every instance where I want the new thing, but this comes up every few days for me, and we'd just never accept being blind like this in any proper software project.